src/modules/TeamsList/components/Teams/components/TeamsHeader/TeamsHeader.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from 'react';
+import {
+  TeamsHeaderStyled,
```

Comment:
you can not add Styled if you do not want

Reply:
Then the components will have the same names